resources/linux/debian/control.in

```diff
@@ -1,6 +1,6 @@
 Package: <%= appFileName %>
 Version: <%= version %>
-Depends: git, libgtk-3-0 (>= 3.9.10), libgcrypt20, libnotify4, libxtst6, libnss3 (>= 2:3.22), libglib2.0-bin | kde-cli-tools | kde-runtime, xdg-utils, libx11-xcb1, libxss1, libxkbfile1, libcurl3 | libcurl4
+Depends: git, libgcrypt20, libgtk-3-0 (>= 3.9.10), libnotify4, libnss3 (>= 2:3.22), libglib2.0-bin | kde-cli-tools | kde-runtime, xdg-utils, libx11-xcb1, libxcb-dri3-0, libxss1, libxtst6, libxkbfile1, libcurl3 | libcurl4
```

Comment:
Do we actually need `libX11-dri-3-0` or is it `libX11-xcb` like on Fedora/openSUSE?

Maybe `libX11-xcb1`? See: https://packages.ubuntu.com/hirsute/amd64/libx11-xcb1/filelist

Reply:
Okay, I see `libxcb-dri3-0` is actually needed, from some testing in Docker.